src/main.rs

```diff
@@ -226,16 +226,29 @@ fn inner_main() -> Result<ExitCode> {
     };
 
     // Reconcile configuration from pyproject.toml and command-line arguments.
-    let exclude: Vec<FilePattern> = cli
-        .exclude
-        .iter()
-        .map(|path| FilePattern::from_user(path, project_root.as_ref()))
-        .collect::<Result<_>>()?;
-    let extend_exclude: Vec<FilePattern> = cli
-        .extend_exclude
-        .iter()
-        .map(|path| FilePattern::from_user(path, project_root.as_ref()))
-        .collect::<Result<_>>()?;
+    let exclude: globset::GlobSet = {
+        let mut ret = globset::GlobSetBuilder::new();
+        for x in cli
+            .exclude
+            .iter()
+            .map(|path| from_user(path, project_root.as_ref()))
```

Comment:
I think if we're going to `for`-loop here, maybe just make this part of the body, rather than doing a comprehension with the loop too. What do you think?

Reply:
Absolutely, the map was just an artifact of old code, fixed in 708d408 and further improved by cc32661